platform/ABObjectApi.js

```diff
@@ -1,4 +1,165 @@
+const crypto = require("crypto");
+
 const ABObjectApiCore = require("../core/ABObjectApiCore");
 
+const CRYPTO_ALGORITHM = "aes-256-gcm";
+const KEY_LENGTH = 32;
+const VI_LENGTH = 16;
+
 module.exports = class ABObjectApi extends ABObjectApiCore {
+   /**
+    * migrateCreate
+    * verify that a table for this object exists.
+    * @param {ABUtil.reqService} req
+    *        the request object for the job driving the migrateCreate().
+    * @param {knex} knex
+    *        the Knex connection.
+    * @return {Promise}
+    */
+   async migrateCreate(req, knex) {
+      const createTasks = [];
+      const modelKey = this.AB.objectKey().model();
+      const modelSecret = this.AB.objectSecret().model();
```

Comment:
Are these new tables? I assume you will update the factory? And migration manager?

Reply:
Thank for your remind.

I added the initialize those object table script here.
https://github.com/digi-serve/ab_runtime/pull/153

But I miss to create this PR.